docs/metadata/lineage.md

```diff
@@ -8,6 +8,61 @@ tags:
   - lineage
 ---
 
-<div class="video-container">
-<iframe src="https://www.youtube.com/embed/ErLWcWAfi8U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
+## What is Lineage?
+
+Lineage tell us about the life cycle of data. Data lineage is the process of understanding, recording, and visualizing
+data as it flows from source to destination. This includes all transformations the data underwent along the way.
+
+## Why is it important?
+
+Just knowing the source of a particular data set is not always enough to understand its importance,
+perform error resolution, understand process changes, and perform system migrations and updates.
+
+Knowing how data is updated and using what transformations, improves overall data quality.
+Also, it allows data custodians to ensure the integrity and confidentiality of data is protected throughout its lifecycle.
+
+Data lineage allows companies to:
+
+1. Track errors in data processes.
+2. Improve overall data quality.
+3. Implement process changes and system migrations with lower risk and more confidence.
+4. Combine data discovery with a comprehensive view of metadata.
+5. Improve overall data governance.
+
+## Navigation
+
+1. Directly from `pipeline tab` in `metadata` by clicking on button as shown in image below.
+2. Using the `Lineage Search` option from the left side pane.
+
+![How to Open Lineage](img/lineage-open-from-metadata.png)
+
+:::info
+Lineage is always computed on-demand directly on the Git code. Therefore, you can do experimental changes in branch and see how it
+will affect the overall lineage and rectify errors if any.
+:::
+
+## Lineage View
+
+![Lineage View](img/lineage-column-level-view.png)
+
+`Browse Datasets` option from right-hand side can be used to search/select the column/entity for which lineage needs to be computed.
+
+![Browse Datasets](img/lineage-browse-dataset.png)
+
+Zoom-in toggle for a particular Pipeline/Dataset can be used to better understand Pipeline/Dataset.
+
+### Pipeline Zoom-In View
+
+This would give code level info around all the components present in the Pipeline.
+
+:::info
+Please select a particular component as shown in below image, to get code level view.
```

Comment:
"as shown in below image, to get code level view" -> "as shown below to get the code-level view"